src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -182,8 +197,11 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
                                             configmap_cluster_name).agent_public_key_certificate
                 except Exception as e:  # pylint: disable=broad-except
                     utils.arm_exception_handler(e, consts.Get_ConnectedCluster_Fault_Type, 'Failed to check if connected cluster resource already exists.')
-                cc = generate_request_payload(configuration, location, public_key, tags, kubernetes_distro, kubernetes_infra)
+                cc = generate_request_payload(configuration, location, public_key, tags, kubernetes_distro, kubernetes_infra, enable_private_link, private_link_scope_resource_id)
                 cc_response = create_cc_resource(client, resource_group_name, cluster_name, cc, no_wait).result()
+                # Disabling cluster-connect if private link is getting enabled
+                if enable_private_link == "true":
```

Comment:
would be good to inform cx about disabling cluster connect in rePUT.

Reply:
The customer will get the prompt message in re-put as well